api/notification-service.yaml

```diff
@@ -254,6 +281,23 @@ components:
           description: Schedule these mesages to go out at the time specified by this UNIX timestamp
           format: int64
 
+    SendSMSRequest:
+      type: object
+      required:
+        - recipient
+        - message
+      properties:
+        recipient:
```

Comment:
Could you indicate that the recipient is a phone number either by adding a description or by changing the name to something like `recipientPhoneNumber`? Or if it's possible for a recipient to be something other than a phone number, give some indictation here so the consumer knows what this is supposed to contain.